app/main.py

```diff
@@ -1 +1,12 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if len(command.split()) != 3:
+        return "Command must be exactly 3 characters"
+    command, initial_file, final_file = command.split(" ")
```

Comment:
There is no need to explicitly define space as a delimiter, it is a default option
```suggestion
    command, initial_file, final_file = command.split()
```